server/views/pages/goal/create/newIndex.njk

```diff
@@ -0,0 +1,190 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "create-goal" %}
+{% set pageTitle = "Create goals with prisoner" %}
+
+{% block beforeContent %}
+  {% include "../../../partials/breadcrumb.njk" %}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: 'There is a problem',
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">Create goals with {{ prisonerSummary.firstName | safe }} {{ prisonerSummary.lastName | safe }}</h1>
+
+      <h2 class="govuk-heading-s">Add goals</h2>
+      <p class="govuk-hint">
+        Goals should be SMART, meaning 'specific', 'measurable', 'achievable', 'relevant' and 'time-bound'. SMART goals allow progress to be better measured and evaluated.<br>
+        You can add the steps to achieve the goal on the next page.
+      </p>
+
+      <form class="form" method="post" novalidate="">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+        <input type="hidden" name="prisonNumber" value="{{ form.prisonNumber }}" />
```

Comment:
The need for `stepNumber` hidden field is unclear at the moment. I understand where it has come from (the update view), and we _might_ need it?, but I'm not sure until we have designed the process and the form object a little better.
For the time being can you remove it please? (though you know we'll probably end up adding it back in again in a couple of weeks! 🤣 )